ui-src/components/LibraryError.tsx

```diff
@@ -0,0 +1,39 @@
+import { Banner, Button, IconWarning32, Link } from '@create-figma-plugin/ui';
+
+import { useFigmaContext } from '@ui/context';
+
+import { Stack } from './Stack';
+
+export const LibraryError = () => {
+  const { reload, cancel, error } = useFigmaContext();
+
+  if (!error) return null;
+
+  return (
+    <Stack space="small">
+      <Stack space="xsmall">
+        <Banner icon={<IconWarning32 />} variant="warning">
+          Oops! It looks like there was an <b>error generating the export file</b>. Please open an
+          issue in our{' '}
+          <Link
+            href="https://github.com/penpot/penpot-exporter-figma-plugin/issues"
+            target="_blank"
+            rel="noreferrer"
+            style="color: black; text-decoration-line: underline;"
```

Comment:
It looks a bit strange having to add custom styles for this component.

Reply:
Apparently there's only Default, New Windows and Full Width options for this component: https://yuanqing.github.io/create-figma-plugin/storybook/?path=/story/inline-text-link--default